Sources/Testing/ABI/EntryPoints/EntryPoint.swift

```diff
@@ -547,6 +566,22 @@ public func configurationForEntryPoint(from args: __CommandLineArguments_v0) thr
   configuration.exitTestHandler = ExitTest.handlerForEntryPoint()
 #endif
 
+  // Warning issues (experimental).
+  if args.isWarningIssueRecordedEventEnabled == true {
+    configuration.eventHandlingOptions.isWarningIssueRecordedEventEnabled = true
+  } else {
+    switch args.eventStreamVersion {
```

Comment:
_For now_, until we've proposed the change list for ABIv1, I think we should just leave this switch out. It'll be easy to add back in v1 (actually, there's a whole different way we can do it that may be more scalable, I'll discuss with you in DMs.)

I say this because somebody passing `1` here is either from the future or is providing what is currently invalid input.

Reply:
See also my comment here: https://github.com/swiftlang/swift-testing/pull/931/files#r1944900283

I realize I'm being confusing, sorry.

Reply:
Replied there, and in the mean time we need this so the tests can opt-in.

Reply:
Aren't the tests passing `isWarningIssueRecordedEventEnabled`?

Reply:
This `switch` case doesn't actually _enable_ the feature for ABI**v1**, it ensures that it remains _disabled_ for ABI**v0**. We definitely want that safeguard, even if we decide not to enable this feature in ABIv1.

Reply:
As of right now, it's dead code since the default value is always `false` though.